corehq/apps/geospatial/views.py

```diff
@@ -435,3 +439,110 @@ def get_users_with_gps(request, domain):
     ]
 
     return json_response({'user_data': user_data})
+
+
+@method_decorator(toggles.GEOSPATIAL.required_decorator(), name="dispatch")
+class CasesReassignmentView(BaseDomainView):
+    urlname = "reassign_cases"
+    MAX_REASSIGNMENT_REQUEST_CASES = 100
+    SYNC_CASES_UPDATE_THRESHOLD = 1000
+    ASYNC_CASES_LIMIT = 5000
+
+    def post(self, request, domain, *args, **kwargs):
```

Comment:
nit: this method got really big and could use some refactor.

Reply:
Addressed in 07efbdcba49e33e994dd4d92538eba51b1c5ca75